### PR TITLE
sale_timesheet_line_exclude 14.0 

### PR DESCRIPTION
**Pull request by PNurm**

### CODE REVIEW

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Please follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Comment by PNurm** [comment]
Apologies, I was looking for the guidelines but I must've missed that, I will go over the module and do the changes as requested.

**Comment by francesco-ooops** [comment]
hi @PNurm , any plans to finish this porting?

File: sale_timesheet_line_exclude/tests/test_sale_timesheet_line_exclude.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                "task_id": task.id,
+                "name": "Entry #1-2",
+                "unit_amount": 1,
+                "employee_id": employee.id,
+                "exclude_from_sale_order": False,
+            }
+        )
+
+        self.assertEqual(timesheet1.timesheet_invoice_type, "billable_time")
+        self.assertEqual(timesheet2.timesheet_invoice_type, "billable_time")
```
Comment (jappi00) [inline sale_timesheet_line_exclude/tests/test_sale_timesheet_line_exclude.py:132]: Travis have a problem here. I do not understand the reason and don't get behind it.

Does anybody have a hint here?

**Comment by jappi00** [comment]
Travis had a problem with the linked test. I do not get behind it. Maybe the default of the field is True?

**Comment by jado95** [comment]
> Travis had a problem with the linked test. I do not get behind it. Maybe the default of the field is True?
@jappi00 
Hi, Maybe error is this:

```
2022-02-01 06:13:36,397 7366 ERROR openerp_test odoo.addons.sale_timesheet_line_exclude.tests.test_sale_timesheet_line_exclude: FAIL: TestSaleTimesheetLineExclude.test_1
Traceback (most recent call last):
File "/home/travis/build/OCA/timesheet/setup/sale_timesheet_line_exclude/odoo/addons/sale_timesheet_line_exclude/tests/test_sale_timesheet_line_exclude.py", line 131, in test_1
self.assertEqual(timesheet1.timesheet_invoice_type, "billable_time")
AssertionError: 'non_billable' != 'billable_time'

- non_billable
+ billable_time
```

**Comment by CRogos** [comment]
@alexey-pelykh I think this can be closed.

**Pull request closed without merging**